_posts/2022-06-08-#22778.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: pr
+date: 2022-06-08
+title: "net processing: Reduce resource usage for inbound block-relay-only connections"
+pr: 22778
+authors: [jnewbery]
+components: ["p2p"]
+host: larryruane
+status: upcoming
+commit: 9db82f1bca0bb51c2180ca4a4ad63ba490e79da4
+---
+
+## Notes
+
+- (These notes describe behavior with or without this PR.)
+
+- We are a `bitcoind` node. Each of our peers is either inbound (our
+  peer initiated the connection) or outbound (we initiated the connection).
+  Once the connection is established, the relationship is symmetric (the
+  protocol works the same in both directions).
+
+- When we initiate an outbound connection, we may decide that we won't
+  relay (forward) transactions to this peer.
+  This peer is designated a `block-relay-only` peer. The peer
+  doesn't know that we're treating it as `block-relay-only`.
```

Comment:
I don't think this description is accurate; we also don't relay addresses to block-relay-only connections.
Maybe pull in a description from and/or link to https://bitcoincore.reviews/19858 and https://bitcoincore.reviews/20726?